### DISCUSSION TITLE
Keyboard shortcut to run command

### DISCUSSION THREAD

**Discussion by marlun** [kovidgoyal/kitty#6266] [Q&A]
I've had the following in my `.inputrc` to be able to write `git checkout <ctrl-k>` and get a fzf  prompt to select between my branches:

    "\C-k": "$(git for-each-ref refs/heads/ --format='%(refname:short)' | fzf)\C-m"

Can I do the same thing with a mapping in kitty to map it to cmd+f for example?

**Comment by kovidgoyal** [comment]
I dont know what you are asking. commands are run inside shells not terminals. If you are asking how to have kitty send arbitrary text to the shell on a keypress search for send_text in the docs.

**Comment by marlun** [comment]
I've been trying with send_text and now managed to do what I wanted with

    map cmd+y send_text all $(git for-each-ref refs/heads/ --format='%(refname:short)' | fzf)\x0d

Thanks!
